src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/FinalLocalVariable/context/defaultContext.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/Test.java b/Test.java
+index 2a731db..eb996ad 100644
+--- a/Test.java
++++ b/Test.java
+@@ -2,12 +2,12 @@ package TreeWalker.coding.FinalLocalVariable;
+ 
+ public class Test {
+     static int foo(int x, int y) {  // violation context
```

Comment:
similar https://github.com/checkstyle/patch-filters/pull/130#issue-455110382, the violation node is IDENT -> x [4:23] and IDENT -> y [4:30] not METHOD_DEF -> METHOD_DEF [4:4].

```
 |--METHOD_DEF -> METHOD_DEF [4:4]
    |   |--MODIFIERS -> MODIFIERS [4:4]
    |   |   `--LITERAL_STATIC -> static [4:4]
    |   |--TYPE -> TYPE [4:11]
    |   |   `--LITERAL_INT -> int [4:11]
    |   |--IDENT -> foo [4:15]
    |   |--LPAREN -> ( [4:18]
    |   |--PARAMETERS -> PARAMETERS [4:19]
    |   |   |--PARAMETER_DEF -> PARAMETER_DEF [4:19]
    |   |   |   |--MODIFIERS -> MODIFIERS [4:19]
    |   |   |   |--TYPE -> TYPE [4:19]
    |   |   |   |   `--LITERAL_INT -> int [4:19]
    |   |   |   `--IDENT -> x [4:23]
    |   |   |--COMMA -> , [4:24]
    |   |   `--PARAMETER_DEF -> PARAMETER_DEF [4:26]
    |   |       |--MODIFIERS -> MODIFIERS [4:26]
    |   |       |--TYPE -> TYPE [4:26]
    |   |       |   `--LITERAL_INT -> int [4:26]
    |   |       `--IDENT -> y [4:30]
    |   |--RPAREN -> ) [4:31]
    |   `--SLIST -> { [4:33]

```